server/internal/graphql/model/models_gen.go

```diff
@@ -82,6 +82,7 @@ type CatalogSelector struct {
 	PatternType []*string `json:"patternType,omitempty"`
 	Userid      []*string `json:"userid,omitempty"`
 	OrgID       []*string `json:"orgID,omitempty"`
+	WorkspaceID []*string `json:"workspaceID,omitempty"`
```

Comment:
Have you written this manually this is an autogenerated file from graphql. Run make graphql after change in the schema